apps/jobs/views.py

```diff
@@ -1260,9 +1248,38 @@ def update_submission(request, challenge_pk):
 
     if request.method == "PATCH":
         submission_pk = request.data.get("submission")
+        submission = get_submission_model(submission_pk)
+        # Submission of submission_input_file for static_dataset_code_upload challenge from submission to evaluation stage
```

Comment:
Change to `Update submission_input_file for is_static_dataset_code_upload submission evaluation`

Reply:
Updated